tools/sdk-testgen/packages/autorest.gotest/test/integrationtest/testAutorest.ts

```diff
@@ -46,7 +46,7 @@ async function runAutorest(readmePath: string, extraOption: string[]) {
     });
 }
 function getExtraOption(outputFolder: string) {
-    return [`--output-folder=${outputFolder}`, '--use=@autorest/go@latest', '--file-prefix="zz_generated_"', '--track2', '--go', '--debug', '--module-version=0.1.0'];
```

Comment:
How about make some RP use '--module-version=0.1.0' and some don't?
Then we can cover both cases.

Reply:
I move the version num to the readme config. For now, two RP will use `0.1.0` and another will use `2.0.0-beta.1`.